embedchain/config/vectordbs/LanceDBConfig.py

```diff
@@ -0,0 +1,39 @@
+import os
+from typing import Dict, List, Optional, Union
+
+from embedchain.config.vectordbs.BaseVectorDbConfig import BaseVectorDbConfig
+from embedchain.helper.json_serializable import register_deserializable
+
+@register_deserializable
+class LanceDBConfig(BaseVectorDbConfig):
+    def __init__(
+        self,
+        table_name: Optional[str],
+        uri: Union[str, List[str]],
+        api_key: Optional[str] =None,
+        region: str = 'us-west-2',
+        host_override: Optional[str] = None,
```

Comment:
Would be nice to also add `**extra_params: Dict[str, any],` to support any new keys that might get added to this in the future

Reply:
done